src/khoj/processor/conversation/anthropic/anthropic_chat.py

```diff
@@ -84,7 +85,12 @@ def extract_questions_anthropic(
         vision_enabled=vision_enabled,
     )
 
-    messages = [ChatMessage(content=prompt, role="user")]
+    messages = []
+
+    if attached_files:
+        messages.append(ChatMessage(content=attached_files, role="user"))
```

Comment:
Should attached_files be passed into `construct_structured_message` similar to how we handle `query_images`?